src/style.css

```
@@ -0,0 +1,146 @@
+html {
+  font-family: roboto, Helvetica, sans-serif;
+}
+
+body,
+h2,
+p {
+  margin: 0;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  align-items: center;
+  box-sizing: border-box;
+
+  padding: 32px 16px 16px;
+
+  background-color: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__picture {
+  margin-bottom: 40px;
+  width: 160px;
+  height: 134px;
+}
+
+.card__product-name {
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  color: #060b35;
+  margin-bottom: 4px;
+}
+
+.card__product-code {
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  color: #616070;
+  margin-bottom: 16px;
+  width: 100%;
+}
+
+.card__review {
+  display: flex;
+  justify-content: space-between;
+  align-items: flex-end;
+  align-self: stretch;
+  
+  margin-bottom: 24px;
+  /* width: 100%; */
+}
+
+/* #region stars */
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  height: 16px;
+  width: 16px;
+  background-repeat: no-repeat;
+  background-position: center;
+  background-image: url(images/star-active.svg);
+  margin-right: 4px;
+}
+
+.stars--0 .stars__star,
+.stars--1 :nth-child(n+2),
+.stars--2 :nth-child(n+3),
+.stars--3 :nth-child(n+4),
+.stars--4 :nth-child(5) {
+  background-image: url(images/star.svg);
+}
+/* #endregion */
+
+.card__reviews {
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+
+  color: #060b35;
+}
+
+.card__price-tag {
+  display: flex;
+  justify-content: space-between;
+  margin-bottom: 16px;
+  /* width: 100%; */
+  align-self: stretch;
+}
+
+.card__price {
+  font-weight: 400;
+  font-size: 12px;
+  line-height: 18px;
+  color: #616070;
+}
+
+.card__cost {
+  font-weight: 700;
+  font-size: 16px;
+  line-height: 18px;
+
+  color: #060b35;
+}
+
+.card__button {
+  display: flex;
+  align-items: stretch;
+  width: 166px;
+  height: 40px;
+  background: #00acdc;
+  border-radius: 5px;
+  border: none;
+
+}
+ 
+.card__button-link {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-grow: 1;
+  font-family: roboto, Helvetica, sans-serif;
+  font-weight: 700;
+  font-size: 14px;
+  line-height: 16px;
+  text-decoration: none;
+  text-transform: uppercase;
+  color: #fff;
+}
+
+.card__button--hover:hover {
+  background: #fff;
+  border: 1px solid #00acdc;
+  border-radius: 5px;
+
+}
+
+.card__button-link--hover:hover {
+  color: #00acdc;
+}
```

чесно кажучи, не зовсім розумію для чого саме тут додаткові модифікатори --hover, ти можеш :hover задавати просто на звичайний клас)
```suggestion
.card__button:hover {
  background: #fff;
  border: 1px solid #00acdc;
  border-radius: 5px;

}

.card__button-link:hover {
  color: #00acdc;
}
```